src/main/java/dwbh/api/services/UserGroupService.java

```diff
@@ -61,14 +68,14 @@ public UserGroupService(
    * Adds an user to a group, if the current user is admin of the group
    *
    * @param currentUser The current user
-   * @param userGroupInput user and group information
+   * @param userGroupIdsInput user and group information
    * @return an instance of {@link Result} (Boolean | {@link java.lang.Error})
    * @since 0.1.0
    */
   @SuppressWarnings("PMD.OnlyOneReturn")
-  public Result<Boolean> addUserToGroup(User currentUser, UserGroupInput userGroupInput) {
-    Group group = groupRepository.getGroup(userGroupInput.getGroupId());
-    User user = userRepository.getUser(userGroupInput.getUserId());
+  public Result<Boolean> addUserToGroup(User currentUser, UserGroupIdsInput userGroupIdsInput) {
```

Comment:
See that you are using the current user just because `isAdmin` needs the user id, and on the other hand you're using the user id from the input to retrieve the user for adding it to the group.

Reply:
More over, why calling `userRepository.getUser(...)` and `userRepository.getGroup(...)` if you only need the ids of both group and user ? It shouldn't be neccessary, the `userGroupRepository.getUserGroup(...)` should resolve that already